src/java/arjdbc/jdbc/JdbcConnectionFactory.java

```diff
@@ -1,45 +1,32 @@
-/***** BEGIN LICENSE BLOCK *****
- * Copyright (c) 2012-2013 Karol Bucek <self@kares.org>
- * Copyright (c) 2006-2010 Nick Sieger <nick@nicksieger.com>
- * Copyright (c) 2006-2007 Ola Bini <ola.bini@gmail.com>
+/*
```

Comment:
Can this just go away?  Major release and is anyone outside really using this?

Reply:
heritage of cherry-picking the backports from support can likely go - will do